pkg/cri/sbserver/container_create.go

```diff
@@ -355,6 +356,9 @@ func (c *criService) volumeMounts(containerRootDir string, criMounts []*runtime.
 		}
 		volumeID := util.GenerateID()
 		src := filepath.Join(containerRootDir, "volumes", volumeID)
+		if !filepath.IsAbs(dst) && goruntime.GOOS != "windows" {
+			dst = filepath.Join("/", dst)
+		}
```

Comment:
maybe a func for this when not goos windows.. something like https://github.com/containerd/containerd/blob/main/remotes/docker/config/hosts.go#L535-L540 

and perhaps a debug out when joining the base to the dst.. should be a very rare occurrence..

Reply:
Following @gabriel-samfira suggestions, the code ended up being different in sbserver and server. So I don't think a function is worth now.

Also, for logging, I need to pass the context to volumeMounts, which doesn't seem worth it, IMHO.

Please re-open if you think it is :)

Reply:
see below comment...